systems/plants/@Manipulator/Manipulator.m

```diff
@@ -54,7 +121,65 @@
         % want to raise for this method, stating that not all outputs were
         % assigned.  (since I can't write dxdot anymore)
         [H,C,B,dH,dC,dB] = obj.manipulatorDynamics(q,v);
+        
+        params = obj.getParamFrame().getPoly;
+        symp = sym('p',[length(params), 1]); %TODO: p is probably a bad idea here.
+        
+        
+        
+        try
+            H = double(H);
```

Comment:
What happens if q and v are of type TaylorVar or TrigPoly for example? I think this try/catch will break that.